### DISCUSSION TITLE
Why make us npm install

### DISCUSSION THREAD

**Discussion by Louis-Stekhoven-Smith** [TailGrids/tailgrids#4] [Q&A]
From what I can see your components should mostly work by just copying and pasting them.

I see there is some custom tailwind stuff that we are adding the the npm install but it is also adding a whole bunch of bloat with your example images etc...

**Comment by Musharofchy** [comment]
Hey Louis, thanks for your query on this. Some of our components are heavily custom style and assets dependent. These are not bloat but required assets which being used while presenting demo data.

Let me know if you need to know anything else

**Comment by osseonews** [comment]
Hey, I had the same question here and agree with Louis. Seems like most components work without any install. Actually, we had a tailwind already set up and don't want to mess with it at all via some new installed plugin. Do you have a list of CSS that comes with the install, so we can just install it manually? I think manual installation is much more preferred for existing tailwind projects than an NPM install. On the components with heavy style, you should have a manual install option and note which CSS needs to be included etc. See how Shadcn does this by providing manual instructions for each component (actually we use Shadcn with your library, but your library has way more components and they look much nicer - still shadcn is good for some functional things, like toasts).

**Reply by Musharofchy** [reply]
Here you go: https://tailgrids.com/docs/installation#using-cdn

**Reply by osseonews** [reply]
Thanks! We purchased a subscription so using that advice. BTW, amazing project here. Any chance you can develop a toast component for premium?
